apps/balancer-tools/src/app/apr/(components)/HeaderEndButton.tsx

```diff
@@ -67,39 +83,28 @@ export default function HeaderEndButton() {
           <span className="font-medium pr-1">Go to pool</span>
         </div>
       </Dialog>
-      <Select
-        placeholder="Select a round"
-        value={selectedRound}
-        onValueChange={(value) => {
-          setSelectedRound(value);
+      <BaseInput
+        value={startAtInput}
+        onChange={(e) => {
+          setStartAtInput((e.target as HTMLInputElement).value);
           router.push(
-            !poolId
-              ? `/apr/round/${value}`
-              : `/apr/pool/${network}/${poolId}/round/${value}`,
+            `/apr/?startAt=${
+              (e.target as HTMLInputElement).value
+            }&endAt=${endAtInput}&`,
           );
         }}
-      >
-        {typeof roundId === "undefined" ? (
-          <SelectItem value="">Select a round</SelectItem>
-        ) : (
-          ""
-        )}
-        <div className="flex flex-col gap-y-1">
-          {ALL_ROUNDS.map((round) => (
-            <SelectItem key={round.value} value={round.value}>
-              <div className="flex gap-x-2 items-center">
-                <Badge color="darkBlue" size="sm">
-                  <div className="flex items-center gap-x-1">
-                    <span className="hidden sm:block">Round</span>
-                    {round.value}
-                  </div>
-                </Badge>
-                <span>{round.label}</span>
-              </div>
-            </SelectItem>
-          ))}
-        </div>
-      </Select>
+      />
+      <BaseInput
+        value={endAtInput}
+        onChange={(e) => {
+          setEndAtInput((e.target as HTMLInputElement).value);
+          router.push(
+            `/apr/?startAt=${startAtInput}&endAt=${
+              (e.target as HTMLInputElement).value
+            }&`,
+          );
+        }}
+      />
```

Comment:
these are to be datepickers, correct? We should have a task ref ensuring we've logged this